python/paddle/utils/dump_config.py

```diff
@@ -37,9 +37,9 @@
     assert isinstance(conf, TrainerConfig_pb2.TrainerConfig)
```

Comment:
from __future__ import print_function

Reply:
This is the usage of python2, which is not encouraged in python3

Reply:
But in Python 2

```
print(a,b)
```

and

```
from __future__ import print_function
print(a, b)
```

is different.

Thi first print is printing a tuple, the second is print a and b.